CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Thrust 1.11.0
+
+## Summary
+
+Thrust 1.11.0 is a minor release providing bugfixes and performance
+enhancements.
+
+It includes a new sort algorithm that provides up to 2x more performance
+from `thrust::sort` when used with certain key types and hardware.
+
+The new `thrust::shuffle` algorithm has been tweaked to improve the randomness
+of the output.
+
+Our CMake package and build system continue to see improvements with
+improved `add_subdirectory` support, installation rules, status messages, and
+other features that make CUB easier to use from CMake projects.
+
+The release includes several other bugfixes and modernizations, and received
+updates from 12 contributors.
+
+## New Features
+
+- NVIDIA/cub#204: New implementation for `thrust::sort` on CUDA when using
+  32/64-bit numeric keys on SM60+. This improved radix sort algorithm provides
+  up to 2x more performance. Thanks for Andy Adinets for this contribution.
+- NVIDIA/thrust#1310, NVIDIA/thrust#1312: Various tuple-related APIs have been
```

Comment:
I like this style of linking; very useful in case we ever migrate the repos again or go to a monorepo.